scss/application.scss

```diff
@@ -3,8 +3,17 @@
 @import 'equalizer';
 
 .navbar-text {
-  font-size: $font-big;
-  margin: 40px 20px 0 0;
+  visibility: hidden;
+}
+
+@media only screen and (min-width: 376px) {
```

Comment:
Separate all the media queries to a separate file called `responsive.scss`, that way it will be a little more readable :smile:

Reply:
Oh, and the media query you will use is:

```scss
@media screen and (max-width: 700px)
```

That way, you only have to change the attributes to organize the elements in responsive and not the other way around